scripts/circleci-run-tests.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -ev
+
+echo "current directory:"
+echo $PWD
+echo "node version:"
+node --version
+echo "npm version:"
+npm --version
+echo "os info:"
+uname -a
+echo "directory:"
+ls -alt
+echo "running tests"
+
+case ${ENVIRONMENT} in
+dev)
+  echo "Testing against dev"
+  export ACCOUNT_SID=${ACCOUNT_SID_DEV}
+  export API_KEY_SECRET=${API_KEY_SECRET_DEV}
+  export API_KEY_SID=${API_KEY_SID_DEV}
+  export REGIONS='us1'
+  export ECS_SERVER=https://ecs.dev-us1.twilio.com
+  export WS_SERVER_INSIGHTS=wss://sdkgw.dev-us1.twilio.com/v1/VideoEvents
+  ;;
+stage)
+  echo "Testing against stage"
+  export ACCOUNT_SID=${ACCOUNT_SID_STAGE}
+  export API_KEY_SECRET=${API_KEY_SECRET_STAGE}
+  export API_KEY_SID=${API_KEY_SID_STAGE}
+  export REGIONS='au1,ie1,us1'
+  export ECS_SERVER=https://ecs.stage-us1.twilio.com
+  export WS_SERVER_INSIGHTS=wss://sdkgw.stage-us1.twilio.com/v1/VideoEvents
```

Comment:
@makarandp0 These are not necessary. The 2.x SDK will build the right urls based on the environment and region.